packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3297,11 +3297,11 @@ export default class Meeting extends StatelessWebexPlugin {
           file: 'meeting/index',
           function: 'setUpLocusInfoSelfListener',
         },
-        EVENT_TRIGGERS.MEETING_BRB_UPDATE
+        EVENT_TRIGGERS.MEETING_BRB_UPDATE,
```

Comment:
Is this event only going to be used when BRB is updated for self? If so, I'm wondering if we should rename it to `EVENT_TRIGGERS.MEETING_BRB_SELF_UPDATE`/`meeting:self:brbUpdate`.